src/coinbase/tests/wallet_test.ts

```diff
@@ -260,20 +260,28 @@ describe("Wallet Class", () => {
       it("should return a Wallet instance", async () => {
         Coinbase.apiClients.wallet!.createWallet = mockReturnValue({
           ...VALID_WALLET_MODEL,
+          network_id: Coinbase.networkList.BaseMainnet,
           server_signer_status: ServerSignerStatus.PENDING,
         });
         Coinbase.apiClients.wallet!.getWallet = mockReturnValue({
           ...VALID_WALLET_MODEL,
+          network_id: Coinbase.networkList.BaseMainnet,
           server_signer_status: ServerSignerStatus.ACTIVE,
         });
         Coinbase.apiClients.address!.createAddress = mockReturnValue(newAddressModel(walletId));
 
-        wallet = await Wallet.create();
+        wallet = await Wallet.create({
+          networkId: Coinbase.networkList.BaseMainnet,
+        });
         expect(wallet).toBeInstanceOf(Wallet);
         expect(wallet.getServerSignerStatus()).toBe(ServerSignerStatus.ACTIVE);
+        expect(wallet.getNetworkId()).toBe(Coinbase.networkList.BaseMainnet);
         expect(Coinbase.apiClients.wallet!.createWallet).toHaveBeenCalledTimes(1);
         expect(Coinbase.apiClients.wallet!.getWallet).toHaveBeenCalledTimes(2);
         expect(Coinbase.apiClients.address!.createAddress).toHaveBeenCalledTimes(1);
+        expect(Coinbase.apiClients.wallet!.createWallet).toHaveBeenCalledWith({
+          wallet: { network_id: Coinbase.networkList.BaseMainnet, use_server_signer: true },
+        });
```

Comment:
server signer cannot be used with Base Mainnet. @alex-stone is working on adding feature gating by network. In the meantime, we should not write tests that violate.

Reply:
+1

Note: this is currently handled on the server-side and will return the appropriate error, but +1 to test setups that match the behavior we will have in prod

Reply:
Makes sense, updated